src/components/staking/PageValidator.vue

```diff
@@ -79,7 +79,11 @@
         <TmBtn
           id="undelegation-btn"
           class="undelegation-btn"
-          :disabled="delegation.amount === 0 && !isInactiveValidator"
+          :disabled="
+            delegation.amount === 0 &&
+            !isInactiveValidator &&
```

Comment:
@Bitcoinera what is this again? I think we need to rename/refactor this. I can't read this.